### PR TITLE
[Docs] Actualize PlatformIO installation instructions

### PR DESCRIPTION
**Pull request by valeros**
This PR updates PlatformIO doc page with extended info on using stable and upstream versions.

cc @pedrominatel @ivankravets

### CODE REVIEW

**Comment by pedrominatel** [comment]
Thank you @valeros for your contribution.

File: docs/source/installing.rst [hunk 1]
```diff
@@ -71,48 +71,70 @@ Installing using PlatformIO
...
+
+Using the stable code
+*********************
+
+.. note::
+   A detailed overview of supported development boards, examples and frameworks can be found on `the official Espressif32 dev-platform page <https://registry.platformio.org/platforms/platformio/espressif32>`_ in the PlatformIO Registry.
+
+The most reliable and easiest way to get started is to use the latest stable version of the ESP32 development platform that passed all tests/verifications and can be used in production.
+
+Create a new project and select one of the available boards. You can change after by changing the *platformio.ini* file.
```
Comment (ivankravets) [inline docs/source/installing.rst:94]: Could we replace `*platformio.ini*` with 

```
[platformio.ini](https://docs.platformio.org/en/latest/projectconf/index.html)?
```

Comment (valeros) [reply docs/source/installing.rst]: Added.

**Comment by me-no-dev** [comment]
@pedrominatel feel free to merge this when it's ready

**Review by pedrominatel — APPROVED** [review]
LGTM

**Pull request merged**